firestore-bigquery-export/scripts/gen-schema-view/src/index.ts

```diff
@@ -41,52 +43,80 @@ const validateInput = (value: any, name: string, regex: RegExp) => {
   return true;
 };
 
+function collect(value, previous) {
+  return previous.concat([value]);
+}
+
+program
+  .name("gen-schema-views")
+  .option("--no-interactive", "Parse all input from command line flags instead of prompting the caller.", false)
+  .option("-p, --project-id <project-id>", "Firebase Project ID for project containing Cloud Firestore database.")
```

Comment:
Should those be consistent with names in the questions?

`project-id` vs `project`. I am happy with either.

Reply:
I have them different now because the names that are specified in `questions` need to match the names of the variables that the input is captured in (https://github.com/firebase/extensions/pull/75/files#diff-fde3bc7b28048e80c1d9ed7c18fe1f47R109-R114). So if the names were consistent, then they would shadow the variables I'm actually passing to the schema generator class: https://github.com/firebase/extensions/pull/75/files#diff-fde3bc7b28048e80c1d9ed7c18fe1f47R93-R95.

What do you think of changing the names of the questions inputs to `<name>Input` (i.e. projectId => projectIdInput)?